src/fparser/two/Fortran2003.py

```diff
@@ -8615,6 +8619,7 @@ def match(string):
                     "ACTION",
                     "ASYNCHRONOUS",
                     "BLANK",
+                    "CONVERT",
```

Comment:
Could you please add CONVERT as an optional extension as we do for the other extensions (see utils.py)

Reply:
Oh yes, I'd forgotten about that. Apologies.